essay_evaluator.py

```diff
@@ -1,15 +1,24 @@
+from dis import dis
 from typing import List
+from discourse_evaluator import DiscourseEvaluator
 
 from discourse_recognizer import DiscourseRecognizer
 from discourse_classifier import DiscourseClassifier
 
 
 class EssayEvaluator:
-    def __init__(self, discourse_recognizer,  discourse_classifer, discourse_evaluator, 
-                 *args, **kwargs) -> None:
-        pass
+    def __init__(
+        self, 
+        discourse_recognizer: DiscourseRecognizer,
+        discourse_classifer: DiscourseClassifier, 

```

Comment:
Looks like we don't need the DiscourseClassifier here, my bad. Please remove this.